h/presenters.py

```diff
@@ -128,14 +125,13 @@ def title(self):
 
         """
         document_ = self.annotation.document
-        if document_:
-            title = document_.title
-            if title:
-                # Convert non-string titles into strings.
-                # We're assuming that title cannot be a byte string.
-                title = text_type(title)
+        title = document_.title
```

Comment:
`annotation.document` can be `None` with the new PG setup. We should probably add a check for this.